hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py

```diff
@@ -56,32 +56,34 @@ def validate_jwt(token: str):
     return {"isAuthorized": False, "context": {"AuthInfo": "JWTTokenCheck"}}
 
 
-def validate_access_token(token: str):
-    response = {"isAuthorized": False, "context": {"AuthInfo": "ServiceAccessToken"}}
+@functools.lru_cache(maxsize=10)
```

Comment:
10 is kind of arbitrary. Perhaps document whey we do not expect this to be a super large number..

Reply:
Added comment with reasoning.